docs/guides/js-waku/run-waku-nodejs.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Run @waku/sdk in a NodeJS Application"
+---
+
+While the `@waku/sdk` package is primarily designed for browser environments, you can use it in a NodeJS application. However, there are certain limitations and considerations to keep in mind. This guide provides a comprehensive overview of using `@waku/sdk` in NodeJS.
+
+## Limitations
+
+### API Compatibility
+
+`@waku/sdk` prioritises browser compatibility, avoiding NodeJS APIs for simpler bundling. This design choice enhances browser API compatibility but sacrifices NodeJS optimisation. While many browser APIs work in NodeJS, they might need better optimisation.
+
+### Protocol Implementation
+
+`@waku/sdk` focuses on the client side of the [Request/Response protocol](/overview/concepts/network-domains#requestresponse-domain). We'll have to replicate all the functionalities added to [nwaku](/guides/run-nwaku-node) to implement extra features.
+
+### Codebase Complexity
+
+Balancing browser and NodeJS compatibility while maintaining simplicity adds complexity. A potential fix is making a NodeJS package that wraps `@waku/sdk` for smoother NodeJS development.
+
+### Browser-Specific Logic
+
+Certain features in `@waku/sdk` are tailored for browsers and might not translate seamlessly to NodeJS. For example, [Relay protocol](/overview/concepts/protocols#relay) in browsers requires unique [transport methods](/overview/concepts/transports) such as `WebTransport` or `WebRTC`, which might not align with NodeJS networking capabilities.
```

Comment:
I would say instead that for the browser, only websocket is supported (in general, not even specific to relay) because TCP is not supported in the browser, whereas a NodeJS application would benefit in using TCP.